### PR TITLE
feat(PE-3587): edit ttl seconds

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx [hunk 1]
```diff
@@ -235,6 +235,13 @@ function TransactionWorkflow({
...
               };
             }
+            case PDNT_INTERACTION_TYPES.SET_TTL_SECONDS: {
+              return {
+                1: { title: 'Confirm TTL Seconds', status: 'pending' },
+                2: { title: 'Deploy TTL Seconds Change', status: '' },
+                3: { title: 'Complete', status: '' },
+              };
+            }
             default:
```
Comment (dtfiedler) [inline src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx:245]: still my opinon we put this switch case into an object in a `constants` file

File: src/components/modals/ManagePDNTModal/ManagePDNTModal.tsx [hunk 2]
```diff
@@ -319,6 +320,20 @@ function ManagePDNTModal() {
...
+                                  PDNT_INTERACTION_TYPES.SET_TTL_SECONDS
+                                ? mapTransactionDataKeyToPayload(
+                                    CONTRACT_TYPES.PDNT,
+                                    row.interactionType,
+                                    [
+                                      '@',
+                                      pdntState!.records['@'].transactionId
+                                        .length
+                                        ? pdntState!.records['@'].transactionId
+                                        : STUB_ARWEAVE_TXID,
```
Comment (dtfiedler) [inline src/components/modals/ManagePDNTModal/ManagePDNTModal.tsx:333]: i'm not a fan of stubbing the tx id

Comment (atticusofsparta) [reply src/components/modals/ManagePDNTModal/ManagePDNTModal.tsx:333]: niether am I but its required per contract code else we cannot se the ttl seconds 
image

File: src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx (hunk 1 above)
Comment (atticusofsparta) [reply src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx]: moved this the transaction data object and created an accessor util

**Pull request closed without merging**